common/cpp/src/google_smart_card_common/pp_var_utils/extraction.h

```diff
@@ -264,8 +257,10 @@ inline bool IsVarArraySizeValid(
     size_t expected_size,
     std::string* error_message) {
   if (var.GetLength() != expected_size) {
-    *error_message = FormatBoostFormatTemplate(
-        kErrorWrongArraySize, expected_size, var.GetLength());
+    *error_message = FormatPrintfTemplate(
+        "Expected an array of size %zu, instead got: %" PRIu32,
```

Comment:
%zu -> %PR... ?

Reply:
Ack - the first parameter has the |size_t| type, and for it, I believe, the "zu" specifier should be used. The "PR..." thing is mainly for types with fixed size.